tests/import_test.go

```diff
@@ -0,0 +1,30 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+)
+
+const (
+	testSuiteName   = "Importer Test Suite"
+	namespacePrefix = "importer"
+)
+
+var _ = Describe(testSuiteName, func() {
+	f := framework.NewFramework(namespacePrefix)
+
+	Describe("Creating standard PVC should not trigger importer", func() {
+		It("PVC should remain empty after being created", func() {
+			pvc, err := f.CreatePVCFromDefinition(f.Namespace.GetName(), framework.NewPVCDefinition("no-import", "1G", nil, nil))
+			Expect(err).ToNot(HaveOccurred())
+			// Wait a while to see if CDI puts anything in the PVC.
```

Comment:
the concept of "a while" is ambiguous. In most (but not all) cases k8s polls for a state to lessen the need for an unconditional wait. Maybe we need to Poll the executor-pod with a cmd to test the existence of _disk.img_?

Reply:
In almost all other cases we can do that, but this is the negative test case (since I don't have http server yet to try a positive case). The test is basically create a PVC without annotation, and wait a while and check if nothing changed. A while like you said is ambiguous, but how else do you check nothing changed?

Reply:
Fair enough for now.